connector/json-test-connector/src/common.rs

```diff
@@ -0,0 +1,93 @@
+use fluvio::{Offset, FluvioConfig, metadata::topic::TopicSpec};
+use fluvio_connector_package::config::ConnectorConfig;
```

Comment:
we should move this file to a separate crate.  maybe `fluvio-connector-base`  or `fluvio-connector-common` ?
post PR item

Reply:
yes, that was the intention.  I would vote for `fluvio-connector-common`  name.

Reply:
sounds good